sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/models/SchemaFormat.java

```diff
@@ -34,4 +34,12 @@ public final class SchemaFormat extends ExpandableStringEnum<SchemaFormat> {
     public static SchemaFormat fromString(String name) {
         return fromString(name.toLowerCase(Locale.ROOT), SchemaFormat.class);
     }
+
+    /**
+     * Creates a new instance of Schema Format.
+     */
+    @Deprecated
+    public SchemaFormat() {
```

Comment:
Add @deprecated Use the {@link #fromString(String)} factory method. javadoc tag.

```suggestion

    /**
     * Creates a new instance of Schema Format.
     * @deprecated Use the {@link #fromString(String)} factory method.
     */
    @Deprecated
    public SchemaFormat() {
```